packages/prismarine/src/block/blocks/Cobweb.ts

```diff
@@ -5,14 +5,13 @@ import Flowable from '../Flowable';
 export default class Cobweb extends Flowable {
     public constructor() {
         super({
-            name: 'minecraft:web', // Called cobweb in the Java Edition
+            name: 'minecraft:cobweb',
             id: BlockIdsType.Cobweb,
             hardness: 4
         });
     }
 
     public getToolType() {
-        // FIXME: add sword
```

Comment:
This isnt correct though. Both shears and sword is correct for cobweb.

Reply:
soo... how do i do it

Reply:
return {BlockToolType.Shears, BlockToolType.Sword}?

Reply:
No, it’d have to be an array. But that isn’t supported yet. Keep it as it was before (with the TODO comment)

Reply:
@BoredSathvik just revert it the removal of the comment and we'll fix it in the future.

Reply:
ok